CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # heroku-buildpack-php CHANGELOG
 
+## v137 (2018-06-26)
+
+### ADD
+
+- PHP/7.1.19 [David Zuelke]
+- PHP/7.2.7 [David Zuelke]
+- ext-mongodb/1.4.4 [David Zuelke]
+- ext-blackfire/1.20.1 [David Zuelke]
+- ext-phalcon/3.4.0 [David Zuelke]
+- ext-pq/2.1.4 [David Zuelke]
+
+### FIX
+
+- New Relic daemon location is broken in PHP INI [David Zuelke]
```

Comment:
The daemon gets started externally; the INI setting isn't actually used (but it's there in case someone wants to modify startup behavior for their own setup), so the "brokenness" can't be tested (and if it could, the test env would need a New Relic endpoint, which, well...)

Reply:
If you want I can add the ability to create addons for https://github.com/heroku/hatchet/.